processing/src/main/java/org/apache/carbondata/processing/loading/model/CarbonLoadModelBuilder.java

```diff
@@ -72,6 +72,28 @@ public CarbonLoadModel build(Map<String, String> options, long UUID, String task
       }
       optionsFinal.put("fileheader", Strings.mkString(columns, ","));
     }
+    String badRecordsFromLoad = options.get("bad_record_path");
+    String badRecordsFromCreate =
+        table.getTableInfo().getFactTable().getTableProperties().get("bad_records_path");
```

Comment:
Move this logic to common place, related to badrecords, this logic repeated so many times